test/unit/file_version/test_file_version.py

```diff
@@ -59,6 +59,19 @@ def test_format_ls_entry(self):
         )
         assert expected_entry == file_version_info.format_ls_entry()
 
+    @pytest.mark.apiver(from_ver=2, to_ver=3)
+    def test_upload_with_time_attribute_v2_v3(self):
```

Comment:
this test should not be restricted so much. While it might need to start from v2, it should probably not end on v3 unless we plan to write a different test / change the behavior in v4.